docker/docker.go

```diff
@@ -84,12 +84,6 @@ func main() {
 
 	setDefaultConfFlag(flTrustKey, defaultTrustKeyFile)
 
-	// Regardless of whether the user sets it to true or false, if they
-	// specify --tlsverify at all then we need to turn on tls
-	if flag.IsSet("-tlsverify") {
```

Comment:
If you look at the help for --tls you'll see this:
```
  --tls=false                          Use TLS; implied by --tlsverify
```
which, to me, means --tlsverify will turn on --tls no matter what I set --tlsverify to.